src/styles/blocks/card.scss

```
@@ -0,0 +1,97 @@
+$main-color: #00acdc;
+$text-accent-color: #060b35;
+$text-secondary-color: #616070;
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: $text-secondary-color;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background: #fff;
+
+  .card__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
```

```suggestion
.card {
  box-sizing: border-box;
  width: 200px;
  border: 1px solid #f3f3f3;
  border-radius: 5px;
  padding: 32px 16px 16px;
  background: #fff;

  &__image {
    display: block;
    margin: 0 auto 40px;
    width: 160px;
    height: 134px;
  }
```

Use SASS nesting like this in **ALL THE CASES**